test/test_um2netcdf.py

```diff
@@ -651,6 +662,379 @@ def test_fix_units_do_nothing_no_um_units(ua_plev_cube):
         assert ua_plev_cube.units == orig  # nothing should happen as there's no cube.units
 
 
+@pytest.fixture
+def lat_river():
+    # iris DimCoord imitating UM V7.3s river grid. Must have length 180.
+    lat_river_points = np.arange(-90., 90, dtype="float32") + 0.5
+    lat_river_coord = iris.coords.DimCoord(
+        points=lat_river_points,
+        standard_name=um2nc.LATITUDE
+    )
+    return lat_river_coord
+
+
+@pytest.fixture
+def lon_river():
+    # iris DimCoord imitating UM V7.3s river grid. Must have length 90.
+    lon_river_points = np.arange(0., 360., dtype="float32") + 0.5
+    lon_river_coord = iris.coords.DimCoord(
+        points=lon_river_points,
+        standard_name=um2nc.LONGITUDE
+    )
+    return lon_river_coord
+
+
+@pytest.fixture
+def lat_v_nd():
+    # iris DimCoord imitating imitating the real
```

Comment:
There's a duplicate "imitating" here (PyCharm again).

Reply:
Good catch, fixed up in latest version